mountainduck/versions.md

```diff
@@ -22,11 +24,48 @@ You can open a previous version of a file by choosing *Versions → ... → Quic
 :width: 900px
 ```
 
-### Revert
+**Revert**
 
 You can revert to a previous version of a file by choosing *Versions → ... → Revert*. Wait for the *File Updated* notification which notifies the previous version has been restored.
 
 ```{image} _images/File_Updated_Notification_Windows.png
 :alt: File Updated Notification (Windows)
 :width: 400px
-```
\ No newline at end of file
+```
+
+````
+````{tab} Version 4.12 and later
+
+The *Versions* option within the contect menu is not available anymore. A list of file versions can be viewed within the *Versions* tab of the *Info* window. The files can also be reverted to a choosen version of the list.
+
+**Supported Providers**
+
+The versions feature within the *Info* window is supported for the following protocols:
+
+| **Protocol** | **Revert previous version** | **Open/Quick Look previous version** | **Delete version** |
+| --- | --- | --- | --- |
+| **[S3](../protocols/s3/index.md)** | ✅ | ✅ | ✅ |
+| **[Backblaze B2](../protocols/b2.md)** | ✅ | ✅ | ✅ |
+| **[Google Drive](../protocols/google_drive.md)** | ❌ | ✅ | ✅ |
+| **[Google Storage](../protocols/google_cloud_storage.md)** | ✅ | ✅ | ✅ |
+| **[Microsoft OneDrive](../protocols/onedrive.md)** | ✅ | ✅ | ❌ |
+| **[Microsoft Sharepoint](../protocols/sharepoint.md)** | ❌ | ✅ | ❌ |
+| **[DRACOON](../protocols/dracoon.md)** | ✅ | ❌ | ✅ |
+| **[Dropbox](../protocols/dropbox.md)** | ✅ | ✅ | ❌ |
+
+```{note}
+Using S3 or Backblaze B2, versions will only be displayed if bucket versioning is enabled.
+```
+
+**Enable Versioning**
+
+**Amazon S3**
+
+Versioning can be enabled per bucket in by choosing *Info → S3* in *Finder* on macOS or *File Explorer* on Windows. Alternatively, enable versioning in AWS Console or [Cyberduck](../cyberduck/index.md).
+
+**Backblaze B2**
+
+Versioning can be enabled per bucket in by choosing *Info → B2* in *Finder* on macOS or *File Explorer* on Windows. Alternatively, enable versioning in AWS Console or [Cyberduck](../cyberduck/index.md).
```

Comment:
There is not currently such an option to toggle file versioning for Backblaze B2 connections. By default all new buckets in B2 have file versioning enabled.